### ISSUE TITLE
[IntelliJ-ThirdParty][A11yMAS][Accessibility][Keyboard][MAS2.1.1][IntelliJ][Wipro]: Unable to access the links present under plugin related information in plugins dialog. 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1453]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Navigate and select Settings Menu item under File menu.
3. In Settings dialog  box, navigate and select Plugins.
4. Plugins dialog will open.
5. In plugins dialog Select any particular plugin(Ex: GitHub Integration).
6. After selecting the plugin navigate to plugin information using "Ctrl+tab" and try to access the links under Plugin information with keyboard.

**Expected Result:**
User should be able to access the "Links" under Plugin related information using keyboard.

**Actual Result:**
Keyboard focus is not visible on "Install" button and "Restart Intellij IDEA" in Browse repositories dialog.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)

![links not accessible under plugin related info](https://user-images.githubusercontent.com/22956130/35800419-c0ddd6a6-0a1d-11e8-8e6d-ab676dd3877b.png)

**Comment by Eskibear** [comment]
The dialog is IntelliJ IDEA's own setting page, it's out of the toolkit's scope.

**Comment by Eskibear** [comment]
tracked in https://youtrack.jetbrains.com/issue/IDEA-191938

**Comment by Eskibear** [comment]
* **Importance of experience**: minimal. This is the feature of IntelliJ IDEA itself. It's out of the plugin scope.
* **Quality of experience**: not impacted. This cannot be controlled by the plugin since it is the IntelliJ IDEA's feature. But if the user do want to see the introduction of any IntelliJ plugins, he can view the online docs for IntelliJ. 
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by jdneo** [comment]
The workaround is to go the official page for the plugin. Take an example used in the screenshot, if the user wants to see details information of this release. He can:

1. Go to the plugin homepage, since the `git integration` plugin is IntelliJ's embedded plugin, the user can go to IntelliJ's official repo to find the description contents: https://github.com/JetBrains/intellij-community/blob/c1cf28c0854449911cbe60bd4fa58ee9e3f53f76/plugins/git4idea/src/META-INF/plugin.xml#L6
2. Then he can access the link directly in the browser

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749887
